diff -Naur a/Makefile b/Makefile
--- a/Makefile	2021-09-30 03:58:38.477488463 -0300
+++ b/Makefile	2021-09-30 04:00:08.112483910 -0300
@@ -60,6 +60,17 @@
 LIBNETLINK=../lib/libutil.a ../lib/libnetlink.a
 LDLIBS += $(LIBNETLINK)
 
+ETC_IPROUTE2=\
+	etc/iproute2/ematch_map\
+	etc/iproute2/nl_protos\
+	etc/iproute2/rt_realms\
+	etc/iproute2/group\
+	etc/iproute2/rt_dsfield\
+	etc/iproute2/rt_protos\
+	etc/iproute2/bpf_pinning\
+	etc/iproute2/rt_tables\
+	etc/iproute2/rt_scopes
+
 all: config
 	@set -e; \
 	for i in $(SUBDIRS); \
@@ -91,7 +102,7 @@
 	install -m 0755 -d $(DESTDIR)$(ARPDDIR)
 	install -m 0755 -d $(DESTDIR)$(HDRDIR)
 	@for i in $(SUBDIRS);  do $(MAKE) -C $$i install; done
-	install -m 0644 $(shell find etc/iproute2 -maxdepth 1 -type f) $(DESTDIR)$(CONFDIR)
+	install -m 0644 $(ETC_IPROUTE2) $(DESTDIR)$(CONFDIR)
 	install -m 0755 -d $(DESTDIR)$(BASH_COMPDIR)
 	install -m 0644 bash-completion/tc $(DESTDIR)$(BASH_COMPDIR)
 	install -m 0644 bash-completion/devlink $(DESTDIR)$(BASH_COMPDIR)
