diff -Naur a/Makefile b/Makefile
--- a/Makefile	2021-10-01 02:41:17.000000000 -0300
+++ b/Makefile	2021-10-02 06:53:03.868630254 -0300
@@ -12,16 +12,16 @@
 all: limine-install
 
 install: all
-	install -d "$(DESTDIR)$(PREFIX)/bin"
-	install -s limine-install "$(DESTDIR)$(PREFIX)/bin/"
-	install -d "$(DESTDIR)$(PREFIX)/share"
-	install -d "$(DESTDIR)$(PREFIX)/share/limine"
-	install -m 644 limine.sys "$(DESTDIR)$(PREFIX)/share/limine/"
-	install -m 644 limine-cd.bin "$(DESTDIR)$(PREFIX)/share/limine/"
-	install -m 644 limine-eltorito-efi.bin "$(DESTDIR)$(PREFIX)/share/limine/"
-	install -m 644 limine-pxe.bin "$(DESTDIR)$(PREFIX)/share/limine/"
-	install -m 644 BOOTX64.EFI "$(DESTDIR)$(PREFIX)/share/limine/"
-	install -m 644 BOOTIA32.EFI "$(DESTDIR)$(PREFIX)/share/limine/"
+	install -d "$(DESTDIR)$(BINDIR)"
+	install -s limine-install "$(DESTDIR)$(BINDIR)"
+	install -d "$(DESTDIR)$(DRTDIR)"
+	install -d "$(DESTDIR)$(DRTDIR)/limine"
+	install -m 644 limine.sys "$(DESTDIR)$(DRTDIR)/limine/"
+	install -m 644 limine-cd.bin "$(DESTDIR)$(DRTDIR)/limine/"
+	install -m 644 limine-eltorito-efi.bin "$(DESTDIR)$(DRTDIR)/limine/"
+	install -m 644 limine-pxe.bin "$(DESTDIR)$(DRTDIR)/limine/"
+	install -m 644 BOOTX64.EFI "$(DESTDIR)$(DRTDIR)/limine/"
+	install -m 644 BOOTIA32.EFI "$(DESTDIR)$(DRTDIR)/limine/"
 
 clean:
 	rm -f limine-install limine-install.exe
